### DISCUSSION TITLE
Step by step Catalina to Monterey Upgrate

### DISCUSSION THREAD

**Discussion by j10ra** [daliansky/XiaoMi-Pro#687] [Q&A]
Is there a step-by-step instruction to upgrade mi notebook pro (8 gen) from macOS Catalina to MacOS Monterey?

**Comment by stevezz3** [comment]
@j10ra 1. Have a full backup on Time Machine or other media.
2. Update to the latest OC EFI, which is v1.7.0 for now.
3. Go to Appstore and search Monterey. Click Install.
4. Wait until finish.

**Reply by j10ra** [reply]
Hi @stevezhengshiqi, Sorry I am very new here. I wanted to ask how I can update to the latest OC EFI. Or a link I can follow to step-by-step update the OC EFI

Thank you @stevezhengshiqi.

**Reply by stevezz3** [reply]
The instruction is at here: https://github.com/daliansky/XiaoMi-Pro-Hackintosh#upgrade
If you changed serial numbers, don't forget to pass the serial number to the new config.

**Reply by j10ra** [reply]
@stevezhengshiqi  how do I replace BOOT and CLOVER(or OC) folders?

**Reply by stevezz3** [reply]
1. Backup and Delete OC and BOOT folders.
2. Copy new OC and BOOT folders to your EFI partition.

**Reply by j10ra** [reply]
Thanks @stevezhengshiqi 
I finally understand! EFI partition needs to be mounted first then I was able to backup and replace the boot and oc folders

If anyone needs. here is a link to mount the EFI partition
https://elitemacx86.com/threads/how-to-mount-efi-partition-on-macos.932/

**Reply by stevezz3** [reply]
Thank you for the link. I added a bit more instruction to README to help people better understand.
